### PR TITLE
Add com.jetbrains.GoLand

### PR DESCRIPTION
**Pull request by dennajort**

### CODE REVIEW

**Comment by dennajort** [comment]
bot, build com.jetbrains.GoLand

**Comment by dennajort** [comment]
bot, build com.jetbrains.GoLand

File: com.jetbrains.GoLand.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      - type: file
+        url: https://download.jetbrains.com/go/goland-2020.2.1.tar.gz
+        sha256: 06cf8a7e6926d0aac2136ae1dde014f70ab71758e8b27b49519e8e47ac174d96
+        size: 424237985
+        x-checker-data:
+          type: jetbrains
+          code: PCP
+        dest-filename: goland.tar.gz
+      - type: file
+        path: com.jetbrains.GoLand.appdata.xml
```
Comment (LeandroStanger) [inline com.jetbrains.GoLand.yaml:55]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Comment (dennajort) [reply com.jetbrains.GoLand.yaml]: Ok, didn't fully understood but I renamed the `*.appdata.xml` to `*.metainfo.xml`and put it into `${FLATPAK_DEST}/share/metainfo/` folder

**Comment by dennajort** [comment]
bot, build com.jetbrains.GoLand

File: com.jetbrains.GoLand.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        -a -f png -o "${icon_out}"; install -p -Dm644 "${icon_out}" -t "${FLATPAK_DEST}/share/icons/hicolor/${s}x${s}/apps/";
+        done;
+    sources:
+      - type: file
+        url: https://download.jetbrains.com/go/goland-2020.2.1.tar.gz
+        sha256: 06cf8a7e6926d0aac2136ae1dde014f70ab71758e8b27b49519e8e47ac174d96
+        size: 424237985
+        x-checker-data:
+          type: jetbrains
+          code: PCP
```
Comment (Lctrs) [inline com.jetbrains.GoLand.yaml:52]: GL

File: README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# com.jetbrains.GoLand
+
```
Comment (bilelmoussaoui) [inline README.md:2]: Not sure if this adds any value to what's already on the website & docs?

File: com.jetbrains.GoLand.desktop [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=GoLand
+GenericName=Capable and Ergonomic Go IDE
+Exec=goland.sh
+Terminal=false
+Icon=com.jetbrains.GoLand
+Type=Application
+Categories=Development;IDE;
+X-Desktop-File-Install-Version=0.21
```
Comment (bilelmoussaoui) [inline com.jetbrains.GoLand.desktop:9]:
```suggestion
```

File: README.md (hunk 3 above)
Comment (dennajort) [reply README.md:2]: I'm just copying what has been done for other JetBrains products for flathub.

File: com.jetbrains.GoLand.yaml (hunk 2 above)
Comment (dennajort) [reply com.jetbrains.GoLand.yaml]: Good catch I missed that one.

File: com.jetbrains.GoLand.desktop (hunk 4 above)
Comment (dennajort) [reply com.jetbrains.GoLand.desktop]: Ok removed this line

**Comment by dennajort** [comment]
bot, build com.jetbrains.GoLand

File: README.md (hunk 3 above)
Comment (barthalion) [reply README.md:2]: It should be removed. Each command is covered by regular flatpak documentation.

Comment (dennajort) [reply README.md:2]: Ok, I removed all the parts related to flatpak.

**Comment by dennajort** [comment]
bot, build com.jetbrains.GoLand

File: README.md (hunk 3 above)
Comment (barthalion) [reply README.md:2]: It's even more unnecessary now. No user is going to read this, this is what appdata is for.

Comment (dennajort) [reply README.md:2]: so better just delete the readme ?

Comment (barthalion) [reply README.md:2]: Yes.

Comment (dennajort) [reply README.md]: Ok removed :+1:

**Comment by dennajort** [comment]
bot, build com.jetbrains.GoLand

**Comment by dennajort** [comment]
Also I want to ask something. Because I mainly copy-pasted from other jetbrains repo in flathub, I digged a little bit more about the freeglut/glu/glew part in the yaml file. And I see that it was added in this PR https://github.com/flathub/com.jetbrains.PyCharm-Community/pull/61 .
I think it may be not necessary to add this into this repo. What do you think ?

**Comment by dennajort** [comment]
bot, build com.jetbrains.GoLand

**Comment by dennajort** [comment]
bot, build com.jetbrains.GoLand

**Comment by dennajort** [comment]
bot, build com.jetbrains.GoLand

**Pull request closed without merging**